app/src/main/java/hacking/to/the/gate/GameManager.java

```diff
@@ -99,39 +99,93 @@ public void setSelfJetDest(MotionEvent event){
     }
 
     /**
+     * detect collision between two circle colliders
+     * @param c1 circle collider one
+     * @param c2 circel collider two
+     * @return true if two collider collided, otherwise false
+     */
+    public boolean detectCollision(CircleCollider c1, CircleCollider c2){
+       Log.d("Position","C1: "+c1.getPosition().getPositionX()+" "+c1.getPosition().getPositionY());
+        return Math.pow(
+                c1.getPosition().getPositionX()-c2.getPosition().getPositionX(),
+                2)
+                +Math.pow(
+                c1.getPosition().getPositionY()-c2.getPosition().getPositionY(),
+                2)
+                < Math.pow(c1.getRadius()+c2.getRadius(),2);
+
+    }
+
```

Comment:
Could you please give me more details or stronger reasons?Actually, I don't think CircleCollider class should have this method even if it's not proper to put the method in GameManager,